Homework9: Ekaterina Zavadskaya

KateZavadskaya opened the pull request:
Add done homework9

Docs: hm9_1.py - hm9_3.py

KateZavadskaya committed:
Feat(hm9): add empty files for hm9

Docs: hm9_1.py - hm9_3.py

KateZavadskaya committed:
Feat(hm9): add files with hm9

Add files hm9_1.py - hm9_3.py with done hm9

Docs: hm9_1.py - hm9_3.py

Stml89 commented on hm9_1.py:
```diff
@@ -0,0 +1,30 @@
+    return "".join(a)
+
+
+print(line_with_symbol("abc#d##c78#"))
```
remove it or use assert

KateZavadskaya replied:
Removed "print" in all hm9.

Stml89 commented on hm9_1.py:
```diff
@@ -0,0 +1,30 @@
+
+    """ Function cleans symbols '#' and the symbol before """
+
+    new_line = line[:]
```
You do not need to cope a list, work with `line`

KateZavadskaya replied:
See you, fixed 👍

KateZavadskaya committed:
Fix(comment): fix comment about extra code

Delete extra code line in hm9_1.py, delete extra print at the end of the code in all hm9

Docs: hm9_1.py - hm9_3.py
